KILTprotocol/polkadot-apps
# feat: update app version, add KILT as chain option

## closes KILTProtocol/ticket#538

- merge all changes from upstream project (newest merged commit dda1ec1d Jun 10)
- add KILT dev and prod chain as an option

## How to test:

```
yarn install
yarn start
```

We probably should change the text for the chains. Also our live chain is more like a test chain and the dev chain should maybe not be listed there? Not sure about that.

## Checklist:
